contract-examples/test/contract_native_minter.ts

```diff
@@ -53,11 +53,6 @@ describe("ERC20NativeMinter", function () {
     })
   });
 
-  it("should add contract deployer as owner", async function () {
-    const contractOwnerAddr: string = await contract.owner()
-    expect(owner.address).to.equal(contractOwnerAddr)
-  });
```

Comment:
Same thing, no need to test `OpenZeppelin` logic.

Reply:
Agree